content/en/docs/setup/pick-right-solution.md

```diff
@@ -195,7 +195,7 @@ These solutions are combinations of cloud providers and operating systems not co
 * [Cloud Foundry Container Runtime (CFCR)](https://docs-cfcr.cfapps.io/)
 * [Gardener](https://gardener.cloud/)
 * [Kublr](www.kublr.com/kubernetes.io/setup-hosted-solution)
-* [Kubernetes on Ubuntu](/docs/getting-started-guides/ubuntu/)
+* [Kubernetes on Ubuntu](https://www.ubuntu.com/kubernetes/docs/quickstart)
```

Comment:
curious, what is the difference between this and https://github.com/kubernetes/website/pull/13594/files#diff-5c9cc686aeb2be89db90a896bbd268f2R72 ?

Reply:
Not a lot, apart from I think that the table has changed since the last one. I closed #13594 myself because there was some discussion (within CDK) on the timing and effect of the changes. The docs pointed to are now up to date with 1.14 etc.